erc20-bridge-token/test/BridgeToken.js

```diff
@@ -27,6 +27,26 @@ describe('BridgeToken', () => {
     await ProofConsumer.transferOwnership(BridgeTokenFactory.address);
   })
 
+  async function setMetadata(nearTokenId, tokenProxyAddress) {
+    const metadataProof = require('./proof_template.json');
+    const metadata = createDefaultERC20Metadata(nearTokenId, 1089);
```

Comment:
Please use constants for the block proof height

Reply:
699afa2dcb032eb54349c0516950d007e028b2b1